h2o-hadoop/scripts/docker/common/setup_h2o

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
```

Comment:
I don't think we should have this in the docker image. Providing H2O jar should be a sole responsibility of the caller.